docs/README-release.md

```diff
@@ -0,0 +1,28 @@
+README-release.md
+========
+
+Conventions for Caffeine Releases
```

Comment:
My thought was not such much a "convention" as a step-by-step release procedure that helps to prevent accidental omissions. 

In my other projects I usually print out this file and literally check off each step with a pen as I complete it.

```suggestion
Release Procedure for Caffeine
```